### PR TITLE
Add Troubleshooting entry about deleted clients

### PR DESCRIPTION
**Pull request by nodeg**
# Description

This will add a new troubleshooting entry about traditional clients and how to register them as Salt minion after deleting them.
Furthermore it fixes some whitespaces in the changelog.

# Target branches

Which documentation version does this PR apply to?

- [x] Master (Default)
- [x] Manager-4.2
- [ ] Manager-4.1
- [ ] Manager-4.0

# Links

### CODE REVIEW

File: modules/installation-and-upgrade/pages/troubleshooting.adoc [hunk 1]
```diff
@@ -86,3 +86,25 @@ This duplication can confuse some browsers.
...
 In most browsers, you can do this quickly by pressing kbd:[Ctrl+F5].
+
+
+
+== Registering a traditional client as Salt minion after deleting it
+
+This is not a valid scenario. Normally you migrate a traditional client to a Salt minion without deleting the client.
+Salt automatically detects that you have a traditional client and does the necessary changes itself. But if you somehow
+deleted the traditional client and want to register it as a Salt minion again, you have to do the following steps on the
+client before registering it as Salt minion:
```
Comment (keichwa) [inline modules/installation-and-upgrade/pages/troubleshooting.adoc:97]:
```suggestion
////
Salt automatically detects that you have a traditional client and does the necessary changes itself.
But if you somehow deleted the traditional client and want to register it as a Salt minion again, you have to do the following steps on the client before registering it as Salt minion:
////
```
I think we do not need the above text.  But keeping for reference for the moment, commented.

File: modules/installation-and-upgrade/pages/troubleshooting.adoc [hunk 2]
```diff
@@ -86,3 +86,25 @@ This duplication can confuse some browsers.
 
 This issue is resolved by clearing the cache and reloading the page.
 In most browsers, you can do this quickly by pressing kbd:[Ctrl+F5].
+
+
+
+== Registering a traditional client as Salt minion after deleting it
+
+This is not a valid scenario. Normally you migrate a traditional client to a Salt minion without deleting the client.
```
Comment (keichwa) [inline modules/installation-and-upgrade/pages/troubleshooting.adoc:94]:
```suggestion
It is recommended to migrate a traditional client to a Salt client directly without deleting it first.
For more information about this migration procedure, see xref:xxx

If you deleted the traditional client by accident, proceed as follows:
```

File: modules/installation-and-upgrade/pages/troubleshooting.adoc [hunk 3]
```diff
@@ -86,3 +86,25 @@ This duplication can confuse some browsers.
...
+Salt automatically detects that you have a traditional client and does the necessary changes itself. But if you somehow
+deleted the traditional client and want to register it as a Salt minion again, you have to do the following steps on the
+client before registering it as Salt minion:
+
```
Comment (keichwa) [inline modules/installation-and-upgrade/pages/troubleshooting.adoc:98]:
```suggestion
.Procedure: Registering traditional client as a Salt client
```

File: modules/installation-and-upgrade/pages/troubleshooting.adoc [hunk 4]
```diff
@@ -86,3 +86,25 @@ This duplication can confuse some browsers.
...
+. Remove the following file:
```
Comment (keichwa) [inline modules/installation-and-upgrade/pages/troubleshooting.adoc:99]:
```suggestion
. On the client, remove the following file:
```

File: modules/installation-and-upgrade/pages/troubleshooting.adoc [hunk 5]
```diff
@@ -86,3 +86,25 @@ This duplication can confuse some browsers.
...
++
+----
+/etc/sysconfig/rhn/systemid
+----
++
+. Remove the following package:
```
Comment (keichwa) [inline modules/installation-and-upgrade/pages/troubleshooting.adoc:105]:
```suggestion
. On the client, remove the [package]``zypp-plugin-spacewalk`` package:
```

File: modules/installation-and-upgrade/pages/troubleshooting.adoc [hunk 6]
```diff
@@ -86,3 +86,25 @@ This duplication can confuse some browsers.
...
++
+----
+zypp-plugin-spacewalk
```
Comment (keichwa) [inline modules/installation-and-upgrade/pages/troubleshooting.adoc:108]:
```suggestion
zypper rm zypp-plugin-spacewalk
```

File: modules/installation-and-upgrade/pages/troubleshooting.adoc [hunk 7]
```diff
@@ -86,3 +86,25 @@ This duplication can confuse some browsers.
...
+----
++
```
Comment (keichwa) [inline modules/installation-and-upgrade/pages/troubleshooting.adoc:110]:
```suggestion
. Register the client now as a Salt client.
  For more information, see xref:xxx
```

File: .changelog [hunk 8]
```diff
@@ -4,6 +4,8 @@
 # - Fixed error in Bat section of Upgrade Guide (bsc#1234567)
 # For guidelines: https://en.opensuse.org/openSUSE:Creating_a_changes_file_(RPM)#Changelog_section_.28.25changelog.29
 
+- Add troubleshooting section regarding registering a deleted traditional client
+  as Salt minion.
```
Comment (keichwa) [inline .changelog:8]:
```suggestion
- Documented registering a deleted traditional client as Salt client in
  the troubleshooting section of the Installation and Upgrade Guide.
```

**Review by keichwa — CHANGES_REQUESTED** [review]
Here is my attempt to normalize it more to our style guide.  We still must look up proper xrefs.

**Comment by nodeg** [comment]
Thank you for the extensive review @keichwa.  I committed your suggestions.

File: modules/installation-and-upgrade/pages/troubleshooting.adoc [hunk 9]
```diff
@@ -86,3 +86,30 @@ This duplication can confuse some browsers.
...
+/etc/sysconfig/rhn/systemid
+----
++
+. On the client, remove the [package]``zypp-plugin-spacewalk`` package:
++
+----
+zypper rm zypp-plugin-spacewalk
+----
+. Register the client now as a Salt client.
+  For more information, see xref:xxx
```
Comment (keichwa) [inline modules/installation-and-upgrade/pages/troubleshooting.adoc:115]:
```suggestion
  For more information, see xref:client-configuration:registration-methods-saltbundle.adoc[].
```

File: modules/installation-and-upgrade/pages/troubleshooting.adoc [hunk 10]
```diff
@@ -86,3 +86,30 @@ This duplication can confuse some browsers.
 
 This issue is resolved by clearing the cache and reloading the page.
 In most browsers, you can do this quickly by pressing kbd:[Ctrl+F5].
+
+
+
+== Registering a traditional client as Salt minion after deleting it
+
+It is recommended to migrate a traditional client to a Salt client directly without deleting it first.
+For more information about this migration procedure, see xref:xxx
```
Comment (keichwa) [inline modules/installation-and-upgrade/pages/troubleshooting.adoc:95]:
```suggestion
Salt detects that you have a traditional client and migrates it to the Salt contact method automatically.
```
I could find this describe elsewhere in our documentation.  Thus no xref: here.

File: modules/installation-and-upgrade/pages/troubleshooting.adoc [hunk 11]
```diff
@@ -86,3 +86,30 @@ This duplication can confuse some browsers.
...
+
+If you deleted the traditional client by accident, proceed as follows:
+////
+Salt automatically detects that you have a traditional client and does the necessary changes itself.
+But if you somehow deleted the traditional client and want to register it as a Salt minion again, you have to do the following steps on the client before registering it as Salt minion:
+////
```
Comment (keichwa) [inline modules/installation-and-upgrade/pages/troubleshooting.adoc:101]:
```suggestion
```
I think we can remove this now.

**Review by keichwa — CHANGES_REQUESTED** [review]
thanks for all your merges.  here are now my final suggestions.

File: modules/installation-and-upgrade/pages/troubleshooting.adoc (hunk 11 above)
Comment (nodeg) [reply modules/installation-and-upgrade/pages/troubleshooting.adoc:101]: Agreed.

**Review by keichwa — APPROVED** [review]
now it LGTM!

**Comment by keichwa** [comment]
If you have merge right, please "squash and merge".  If not, I can do it for you!

**Review by 0rnela — APPROVED** [review]

**Comment by nodeg** [comment]
I did a rebase because of conflicts in the changelog.

**Pull request merged**